src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css

```diff
@@ -820,3 +820,141 @@
 .monaco-workbench .part.editor>.content .gettingStartedContainer .hide-category-button:hover {
 	background-color: var(--vscode-toolbar-hoverBackground);
 }
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer {
+	background: var(--vscode-welcomePage-background);
+	color: var(--vscode-foreground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .description {
+	color: var(--vscode-descriptionForeground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .category-progress .message {
+	color: var(--vscode-descriptionForeground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .gettingStartedSlideDetails .gettingStartedDetailsContent > .getting-started-footer {
+	color: var(--vscode-descriptionForeground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .icon-widget {
+	color: var(--vscode-textLink-foreground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .gettingStartedSlideDetails .getting-started-step .codicon-getting-started-step-checked {
+	color: var(--vscode-textLink-foreground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .gettingStartedSlideDetails .getting-started-step.expanded .codicon-getting-started-step-unchecked {
+	color: var(--vscode-textLink-foreground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer button {
+	background: var(--vscode-welcomePage-tileBackground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .gettingStartedSlideCategories .getting-started-category {
+	filter: drop-shadow(2px 2px 2px var(--vscode-welcomePage-tileShadow));
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer button:hover {
+	background: var(--vscode-welcomePage-tileHoverBackground);
+	outline-color: var(--vscode-contrastActiveBorder);
```

Comment:
This is causing me trouble, not sure what the solution is. This color token is null in the dark, and light theme. Meaning that normally in the `registerThemeingParticipant` it wouldn't be set at all, but here it seems to default to white causing a difference in styling. I tried playing around with it for awhile, but don't know a good fix to keep the current style we have now. Should we define this for the other theme types too? That feels unnecessary

Reply:
You can use `var(--name, <fallback>`) like `outline-color: var(--vscode-contrastActiveBorder, inherit)` or so. That is still extra styling but not visible

Reply:
Hmm. That didn't change anything. Is there a good way to debug CSS variables. Maybe I need a set of fresh eyes as I can't find a difference besides that line

Reply:
🤔 I haven't tried it myself but that's how it should work... https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values.

Reply:
The fallback works, it's just there's no way to act like the rule just isn't there if the variable is not defined. I've fallen back to the focus border color which is what would normally be applied anyways if the value isn't set. Just something to be aware of.